accounts/service.go

```diff
@@ -219,6 +240,33 @@ func (s *InterceptorService) Stop() error {
 	return s.store.Close()
 }
 
+// IsRunning checks if the account service is running, and returns a boolean
+// indicating whether it is running or not.
+func (s *InterceptorService) IsRunning() bool {
+	s.RLock()
+	defer s.RUnlock()
+	return s.isEnabled
+}
+
+// RequireRunning checks that the account service is running, and returns an
+// error if it is not.
+//
+// NOTE: The store lock MUST be held as either a read or write lock when calling
+// this method.
+func (s *InterceptorService) requireRunning() error {
```

Comment:
I think we can just use the IsRunning() method instead of this

Reply:
Ah, sorry. I ment the discussion that we need to check if the service is running under the same lock, and not drop the lock to then reacquire it, so we still need two different versions of this function which is why I kept this function before.

But agree that it's confusing with the error, so I changed the PR now to instead include a `isRunningUnsafe` function that doesn't acquire the lock, and use that function instead where it is needed :)